### ISSUE TITLE
Button focus indicator has insufficient color contrast

### ISSUE THREAD

**Issue by Woebin** [JohnXLivingston/peertube-plugin-livechat#729]
**Describe the bug**
This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 1.4.11 (AA) Non-text contrast](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast)

People with reduced vision often have difficulty seeing components and graphics in the interface that do not have high enough contrasts against the background. This includes buttons, form fields, icons, illustrations and diagrams (such as curves and arrows).

The buttons to enter the chat or log in using an external account have custom focus indicators which lack sufficient color contrast against the background.

![Button with keyboard focus, and a contrast checking tool indicating insufficient color contrast.](https://github.com/user-attachments/assets/ee332fae-331f-4ebb-8f39-567b46c6f2ac)

**To Reproduce**
1. Without selecting a screen name, tab through the chat interface until a button receives focus.
2. See that the focus indicator has insufficient color contrast against the background.

**Expected behavior**
Graphical objects must have a contrast value of 3:1 against the background, including custom focus indicators. You can use tools such as [](https://contrast.report/) to measure this.

If you refrain from customizing the focus indicators you don't have to think about this to be compliant, but it's often better to design a custom focus indicator for improved visibility. On the example page the background is light, so making the focus indicator darker would be an intuitive way of solving this issue.

There are many articles that delve further into designing accessible focus indicators, such as [this one](https://www.deque.com/blog/give-site-focus-tips-designing-usable-focus-indicators/) from Deque.

**Server:**
 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: 14.0.0

**Desktop:**
 - OS: MacOS 15.5
 - Browser: Google Chrome Version 137.0.7151.120 (Official Build) (arm64)

**Additional context**
This issue was recorded as part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), contracted by [HowlRound Theatre Commons](https://howlround.com/).
